Helpers/time.js

```diff
@@ -0,0 +1,24 @@
+export function convertTime(state) {
+  let newTime = state.replace(/T.*/, "")
+
+  newTime = newTime.slice(2, newTime.length).split("-").reverse().join("-")
+
+  return newTime
+}
+
+export function completedInDays(createdAt) {
+  let todoAddTime = createdAt.replace(/Created At: /, "")
+
+  todoAddTime = todoAddTime.split("-").reverse().join("-")
+
+  let todoFinishTime = new Date().toISOString().replace(/T.*/, "")
+
+  todoAddTime = `20${todoAddTime}`
+
+  let date1 = new Date(todoAddTime)
+  let date2 = new Date(todoFinishTime)
```

Comment:
Please use `const` here bhai as `date1` and `date2` values did not change but just got used inside the file.

```suggestion
  const date1 = new Date(todoAddTime)
  const date2 = new Date(todoFinishTime)
```